setup.py

```diff
@@ -51,9 +51,8 @@
     description='Packs for Redis modules into a distributable format',
     author='RedisLabs',
     url='https://github.com/redislabs/RAMP',
-    scripts=['packer/module_packer'],
     license='BSD 2-clause',
```

Comment:
return script because we have an executable here and it's not just a library.

The command name should be ramp and the syntax should be:
```
ramp [pack|unpack|verify] ....
```